src/Profiler/DataCollector/BlockDataCollector.php

```diff
@@ -16,45 +16,25 @@
 use Sonata\BlockBundle\Templating\Helper\BlockHelper;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface;
+use Symfony\Component\HttpKernel\DataCollector\DataCollector;
 
 /**
  * Block data collector for the symfony web profiling.
  *
  * @author Olivier Paradis <paradis.olivier@gmail.com>
  */
-class BlockDataCollector implements DataCollectorInterface, \Serializable
+class BlockDataCollector extends DataCollector
```

Comment:
depending on a concrete class instead of an interface? why is this a good idea?

Reply:
Because it's an abstract class provided by the framework which uses exactly this class.